sdk/go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3 // indirect
 	golang.org/x/sys v0.0.0-20190403152447-81d4e9dc473e
 	golang.org/x/text v0.3.1-0.20181227161524-e6919f6577db // indirect
+	google.golang.org/appengine v1.4.0 // indirect
```

Comment:
I saw this change for a while and it feels like everyone is afraid to actually commit it. You wanna do it now? 😄

Reply:
Sure, I hate having phantom go.mod entries that pop up after I pull.